pandapower/opf/PpPmInterface/src/PpPmInterface.jl

```diff
@@ -2,40 +2,39 @@ module PpPmInterface
 
 import JuMP
 import JSON
-import PowerModels
+
+using PowerModels
 using Cbc
 using Ipopt
 using Juniper
 
-try
-    using Gurobi
-catch e
-    if isa(e, LoadError)
-        println("Cannot import Gurobi. That's fine if you do not plan to use it")
-    end
-end
-
-try
-    using KNITRO
-catch e
-    if isa(e, LoadError)
-        println("Cannot import KNITRO. That's fine if you do not plan to use it")
-    end
-end
-
-try
-    using SCIP
-catch e
-    if isa(e, LoadError)
-        println("Cannot import SCIP. That's fine if you do not plan to use it")
-    end
-end
+using Gurobi
 
-include("input/pp_2_pm.jl")
-include("pm_models/run_powermodels.jl")
-include("pm_models/run_powermodels_mn_storage.jl")
-include("pm_models/run_powermodels_ots.jl")
-include("pm_models/run_powermodels_powerflow.jl")
-include("pm_models/run_powermodels_tnep.jl")
+include("input/pp_to_pm.jl")
+export load_pm_from_json, get_model, get_solver
```

Comment:
There is no need to export internal functions, only functions that we need outside the project environment should be exported. BTW, the exported functions should be mentioned after import/use pkgs and before including files.